walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/datasource/network/credentialOperations/FetchContractNetworkOperation.kt

```diff
@@ -6,28 +6,26 @@
 package com.microsoft.walletlibrary.did.sdk.datasource.network.credentialOperations
 
 import com.microsoft.walletlibrary.did.sdk.credential.service.models.contracts.VerifiableCredentialContract
-import com.microsoft.walletlibrary.did.sdk.credential.service.models.serviceResponses.ContractServiceResponse
 import com.microsoft.walletlibrary.did.sdk.credential.service.validators.JwtValidator
 import com.microsoft.walletlibrary.did.sdk.crypto.protocols.jose.jws.JwsToken
 import com.microsoft.walletlibrary.did.sdk.datasource.network.GetNetworkOperation
-import com.microsoft.walletlibrary.did.sdk.datasource.network.apis.ApiProvider
+import com.microsoft.walletlibrary.did.sdk.datasource.network.apis.HttpAgentApiProvider
 import com.microsoft.walletlibrary.did.sdk.util.controlflow.DidInHeaderAndPayloadNotMatching
 import com.microsoft.walletlibrary.did.sdk.util.controlflow.InvalidSignatureException
-import com.microsoft.walletlibrary.did.sdk.util.controlflow.IssuanceException
-import com.microsoft.walletlibrary.did.sdk.util.controlflow.Result
+import com.microsoft.walletlibrary.util.http.httpagent.IResponse
 import kotlinx.serialization.json.Json
-import retrofit2.Response
 
 internal class FetchContractNetworkOperation(
     val url: String,
-    apiProvider: ApiProvider,
```

Comment:
I have left the comment here, since I couldn't leave it on those classes. Could we remove the old api interfaces and classes since they aren't used anymore?